Makefile

```diff
@@ -132,7 +132,7 @@ clean_certs:
 	rm -f db/remotedb/::.crt db/remotedb/::.key
 
 test_libs: gen_certs
-	go test -tags gcc $(PACKAGES)
+	go test -tags clevedb boltdb $(PACKAGES)
```

Comment:
Shouldn't these tags be in quotes? Otherwise it looks like we're trying to run the `boltdb` package ... shuold be `-tags "cleveldb boltdb"`